cloud-optimized-geotiffs/cogs-details.qmd

```diff
@@ -1,28 +1,29 @@
 ---
-title: "Make better GeoTIFFs"
+title: "Advanced COG Details"
 ---
 
-# Make better GeoTIFFs 
+# Advanced COG Details
```

Comment:
I thought about this name too, but realized none of these additional details are COG specific and wanted to make that clear. Can we add a clarifying sentence to the intro that all of this is valid for making GeoTIFF useful not just COGs?

Reply:
👍 we can either leave the title as GeoTIFF and add a sentence noting that everything is valid for COG, or put COG in the title and mention everything is valid for GeoTIFF. Not sure which is less confusing

Reply:
I changed this to `Advanced COG/GeoTIFF Details`, with some wording mentioning that most of the article applies both to COG and to GeoTIFFs